packages/react-core/src/components/Tabs/Tab.tsx

```diff
@@ -39,6 +41,8 @@ export interface TabProps extends Omit<React.HTMLProps<HTMLAnchorElement | HTMLB
   closeButtonAriaLabel?: string;
   /** @beta Flag indicating the close button should be disabled */
   isCloseDisabled?: boolean;
+  /** @beta Actions rendered beside the tab content */
+  action?: React.ReactNode;
```

Comment:
To match how other components have a similar prop named (since it looks like we might intend 1+ actions to be passed in):

```suggestion
  actions?: React.ReactNode;
```